mods/ts/weapons/energyweapons.yaml

```diff
@@ -207,3 +207,13 @@ LaserFence:
 		DebugOverlayColor: FF0000
 		Damage: 99999
 		DamageTypes: FireDeath
+
+Firestorm:
+	ReloadDelay: 16
+	ValidTargets: Ground, Air
+	Warhead@1Dam: SpreadDamage
```

Comment:
We should introduce an `InstantKillWarhead` that cannot be blocked by damage modifiers for this and the laser fences.

Reply:
I had a bit of a hard time to get visual `DamageTypes` to work so it looks a bit unpolished for infantry.